tests/integration/test_fim/test_files/test_basic_usage/test_basic_usage_access_opened_files.py

```diff
@@ -0,0 +1,219 @@
+            brief: Restart the `wazuh-syscheckd` daemon.
+        - wait for scan start:
+            type: fixture
+            brief: Wait for start of initial FIM scan start.
+
+    assertions:
+        - Verify that the file hast been modified renamed.
+        - Verify that the modificaction is done before the initial scan ends.
+    """
+
+    check_apply_test(tags_to_apply, get_configuration["tags"])
+
+    # Wait a few seconds for scan to run on created file.
+    time.sleep(3)
+
+    modify_time = None
+    # Modify the file
+    changed_path = os.path.join(directory_str, "changed_name")
+    try:
+        modify_time = time.time()
+        os.rename(file_path, changed_path)
+        # Assert the file has been changed
+        assert os.path.isfile(changed_path) == True
+    except (OSError, IOError, PermissionError) as error:
+        pytest.fail(f"Could not rename file - Error: {error}")
+
+    # Capture scan end timestamp & assert the file was modified before scan end
+    scan_timestamp = get_scan_timestamp(wazuh_log_monitor)
+    assert modify_time < scan_timestamp
+
+
+@pytest.mark.parametrize("tags_to_apply", [({"ossec_conf"})])
+def test_basic_usage_delete_opened_files(tags_to_apply, get_configuration, configure_environment,
+                                        create_and_delete_file, restart_syscheckd_function, wait_for_scan_start):
+    """
+    description: Check that files that are being scanned by syscheckd daemon
+                 can deleted deleted, and that wazuh is not
+                 blocking the files.
+
+    wazuh_min_version: 4.2
+
+    parameters:
+        - tags_to_apply:
+            type: string
+            brief: Tells which configuration to use.
+        - get_configuration:
+            type: fixture
+            brief: Get configurations from the module.
+        - configure_environment:
+            type: fixture
+            brief: Configure a custom environment for testing.
+        - create_and_delete_file:
+            type: fixture
+            brief: Creates file to be monitored. Cleans enviroment after test.
+        - restart_syscheckd_function:
+            type: fixture
+            brief: Restart the `wazuh-syscheckd` daemon.
+        - wait for scan start:
+            type: fixture
+            brief: Wait for start of initial FIM scan start.
+
+    assertions:
+        - Verify that the file has been deleted.
+        - Verify that the modificaction is done before the initial scan ends.
+
+    """
+
+    check_apply_test(tags_to_apply, get_configuration["tags"])
+
+    # Wait a few seconds for scan to run on created file.
+    time.sleep(3)
+
+    modify_time = None
+    # Delete the file
+    try:
+        os.remove(file_path)
+        modify_time = time.time()
+        # Assert the file has been deleted
+        assert os.path.isfile(file_path) == False
+    except (OSError, IOError, PermissionError) as error:
+        pytest.fail(f"Could not delete file - Error: {error}")
```

Comment:
I think that we could delete the assert because I can note if this file is deleted correctly the assert always be true but when the remove fails the assert never run.

In case of using this suggestion remember to edit the line 74 with import of `remove_file`
```suggestion
    try:
        remove_file(file_path)
        modify_time = time.time()
    except (OSError, IOError, PermissionError) as error:
        pytest.fail(f"Could not delete file - Error: {error}")
```